draft-kazuho-httpbis-priority.md

```diff
@@ -168,37 +168,37 @@ priority = urgency=non-blocking, progressive=?1
 
 # Merging Client- and Server-Driven Parameters {#merging}
 
-It is not always the case that the client has the best view of how the HTTP
-responses should be prioritized.  For example, whether a JPEG image should be
-served progressively by the server depends on the structure of that image file
-- a property only known to the server.
+It is not always the case that the client has the best understanding of how the
+HTTP responses deserve to be prioritized.  For example, use of an HTML document
+might depend heavily on one of the inline images.  Existence of such dependency
+is typically best known to the person that administers the document.
```

Comment:
"person that administers" sounds like an odd turn of phrase to me. I think you were shooting for replacing implementation-oriented "server" but I don't know if this hits the mark. I have no other suggestions though!

Reply:
I agree that "server" is better. Fixed in 21624c7.